opm/autodiff/StandardWellsDense.hpp

```diff
@@ -69,7 +72,9 @@ enum WellVariablePositions {
         template<typename FluidSystem, typename BlackoilIndices>
```

Comment:
IMO, it would be better to pass the type tag as a template parameter instead of hardcoding it. The fluid system and the indices (if this is the BlackOilIndices class from eWoms) can be accessed via the property system:

```c++
typedef typename GET_PROP_TYPE(TypeTag, FluidSystem) FluidSystem;
typedef typename GET_PROP_TYPE(TypeTag, Indices) BlackOilIndices;
```